types/wordpress__blocks/index.d.ts

```diff
@@ -19,6 +20,10 @@ declare module '@wordpress/data' {
     function select(key: 'core/blocks'): typeof import('./store/selectors');
 }
 
+export type AxialDirections = 'horizontal' | 'vertical';
+
+export type CSSDirections = 'top' | 'right' | 'bottom' | 'left';
```

Comment:
I think these are usually named in the singular, and arrays fill the plurality requirement. For example, `CSSDirection[]` is multiple CSS directions. A value of the given type can only inhabit one of the values so while it looks like we're discussing _all_ of them, we're actually saying, "A value of type `AxialDirections` can only be `horizontal` or `vertical`" which reads more awkwardly than "A value of the type `AxialDirection` can only be `horizontal` or `vertical`"

Don't want to block this PR because of that, so if you are willing consider renaming these two into the singular. That being said, I'll approve it in advance anyway.